src/3_api.js

```diff
@@ -301,8 +307,8 @@ Server.prototype.XHRRequest = function(url, data, method, storage, callback, nop
 		if (req.readyState === 4) {
 			utils.addPropertyIfNotNull(utils.instrumentation, brttTag, utils.calculateBrtt(brtt));
 			if (req.status === 200) {
-				if (noparse) {
-					data = req.responseText;
+				if (noParse) {
+					data = req.response;
```

Comment:
Can you explain this change from 'responseText' to 'response'?

Reply:
added explanation